### PR TITLE
[ITEM][SCRIPT] Tizona Lua file

### PR DESCRIPTION
**Pull request by nesstea**
Enjoy.

### CODE REVIEW

File: scripts/globals/items/tizona.lua [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+-- onAdditionalEffect Action
+-----------------------------------
+
+function onAdditionalEffect(player,target,damage)
+    
+	local chance = 10;
+    if (math.random(0,99) >= chance) then
+        return 0,0,0;
+    else
+        local drain = math.floor(utils.clamp(damage, 1, 45) * (math.random(100,200)/1000));
```
Comment (teschnei) [inline scripts/globals/items/tizona.lua]: why the clamp?

**Comment by teschnei** [comment]
lots of tabs that should be replaced with spaces

**Comment by nesstea** [comment]
New server box. Forgot to change that before.

**Comment by nesstea** [comment]
Made the changes.

**Pull request merged**